### PR TITLE
changed common nouns tagged PROPN/NNP(S) outside a named entity as NO…

### PR DESCRIPTION
**Pull request by jheinecke**
As proposed in issue #91 I have changed the UPOS from PROPN to NOUN (and XPOS from NNP(S) to NN(S) of uncapitalized nouns if they are not part of a named entity.

I saw elsewhere the discussion whether the lemma of adjectives in comparative or superlative should be the base form of the adjective (e.g. form: nicest lemma: nice) or not. In my usage of UD_English-EWT I have changed the lemmas of these adjectives. I can provide this modification too

### CODE REVIEW

**Comment by sebschu** [comment]
Hi,

Thanks for the contribution -- this looks great!

However, you'll have to make to changes so that this can be merged in. First, please add all the changes in the files in `not-to-release/sources`. The train/dev/test files are automatically generated from the individual files there and all your changes would be overwritten when we create a new release (see also https://github.com/UniversalDependencies/UD_English-EWT#contributing).

Second, please make a pull request on the **dev** branch. The master branch is protected and only updated when there is a new release.

Thanks,
Sebastian

File: en_ewt-ud-train.conllu [hunk 1]
```diff
@@ -165044,7 +165044,7 @@
 38	the	the	DET	DT	Definite=Def|PronType=Art	39	det	39:det	_
 39	hippies	hippie	NOUN	NNS	Number=Plur	36	nmod	36:nmod:of	_
 40	were	be	AUX	VBD	Mood=Ind|Tense=Past|VerbForm=Fin	41	cop	41:cop	_
-41	caucasians	caucasians	PROPN	NNPS	Number=Plur	34	conj	27:advcl:because|34:conj:and	_
+41	caucasians	caucasian	NOUN	NNS	Number=Plur	34	conj	27:advcl:because|34:conj:and	_
```
Comment (nschneid) [inline en_ewt-ud-train.conllu:165047]: should be PROPN

File: en_ewt-ud-train.conllu [hunk 2]
```diff
@@ -176751,7 +176751,7 @@
 27	us	we	PRON	PRP	Case=Acc|Number=Plur|Person=1|PronType=Prs	24	obl	24:obl:like	_
 28	anglo	anglo	X	AFX	_	30	compound	30:compound	SpaceAfter=No
 29	-	-	PUNCT	HYPH	_	30	punct	30:punct	SpaceAfter=No
-30	saxons	saxons	PROPN	NNPS	Number=Plur	27	appos	27:appos	SpaceAfter=No
+30	saxons	saxon	NOUN	NNS	Number=Plur	27	appos	27:appos	SpaceAfter=No
```
Comment (nschneid) [inline en_ewt-ud-train.conllu:176754]: should be PROPN

File: en_ewt-ud-train.conllu [hunk 3]
```diff
@@ -164988,10 +164988,10 @@
 # text = As the southeast asians and philipinos came over, they added to the cultures.
 1	As	as	SCONJ	IN	_	7	mark	7:mark	_
 2	the	the	DET	DT	Definite=Def|PronType=Art	4	det	4:det	_
-3	southeast	southeast	PROPN	NNP	Number=Sing	4	compound	4:compound	_
-4	asians	asians	PROPN	NNPS	Number=Plur	7	nsubj	7:nsubj	_
+3	southeast	southeast	NOUN	NN	Number=Sing	4	compound	4:compound	_
+4	asians	asian	NOUN	NNS	Number=Plur	7	nsubj	7:nsubj	_
```
Comment (nschneid) [inline en_ewt-ud-train.conllu:164992]: should be PROPN

File: en_ewt-ud-train.conllu [hunk 4]
```diff
@@ -164988,10 +164988,10 @@
...
 5	and	and	CCONJ	CC	_	6	cc	6:cc	_
-6	philipinos	philipinos	PROPN	NNPS	Number=Plur	4	conj	4:conj:and|7:nsubj	_
+6	philipinos	philipino	NOUN	NNS	Number=Plur	4	conj	4:conj:and|7:nsubj	_
```
Comment (nschneid) [inline en_ewt-ud-train.conllu:164994]: should be PROPN; lemma: Filipino; Typo=Yes

**Pull request closed without merging**